### PR TITLE
S04 us11 filter offer

### PR DESCRIPTION
**Pull request by maxime-guichard**

### CODE REVIEW

**Review by Erdnaxela45 — DISMISSED** [review]

File: src/DataFixtures/JobFixtures.php [hunk 1]
```diff
@@ -27,7 +28,11 @@ public function load(ObjectManager $manager)
             $job->setCity($faker->city());
             $job->setDescription($faker->paragraph(3, true));
             $job->setCompany($this->getReference('company_' . rand(1, CompanyFixtures::LOOPNUMBER)));
-            $job->setStudyField($this->getReference('studyField_' . rand(1, StudyFieldFixtures::LOOPNUMBER)));
+            for ($i = 0; $i < count(StudyFieldFixtures::STUDYFIELD); $i++) {
+                $job->setStudyField(
+                    $this->getReference('studyField_' . rand(1, count(StudyFieldFixtures::STUDYFIELD) - 1))
+                );
+            }
```
Comment (LucyMerlier) [inline src/DataFixtures/JobFixtures.php:35]: why use a loop here??

File: src/Entity/FilterStudyField.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+namespace App\Entity;
+
+use App\Entity\StudyField;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class FilterStudyField
+{
+    /**
+     * @Assert\Type("App\Entity\StudyField")
+     */
```
Comment (LucyMerlier) [inline src/Entity/FilterStudyField.php:12]: we actually don't need any assert here => my bad I must have been tired when I told you to do that

File: templates/offer/index.html.twig [hunk 3]
```diff
@@ -6,56 +6,34 @@
...
-        </div>
-    </section>
+	<section class="adminOffre">
+		<div class="container">
+			<h1 class="text-center mt-3">Offres d'emploi</h1>
+			<div class="d-flex justify-content-center">
+            	{{ form_start(form) }}
+                	{{ form_widget(form) }}
+                    <button>Filtrer</button>
+                {{ form_end(form) }}
```
Comment (LucyMerlier) [inline templates/offer/index.html.twig:16]: indentation

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by endricklosange — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**